package.json

```diff
@@ -5,6 +5,7 @@
   "repository": "github:nushell/nushell.github.io",
   "type": "module",
   "scripts": {
+    "update": "nu book/make_docs.nu",
```

Comment:
Interesting that this didn't conflict with the changes in the commit. 
Mhh wouldn't the command reference then be independent of what is in the folder on the repo?

Reply:
I'm not sure what you're asking. The idea with this command is that it assumes whoever runs `npm run update` always has the most recent publicly-released version installed.

Reply:
Ah sorry, I have to admit that I am not really familiar with the build process for github pages. I assumed this would be run on deploy. Having this as a shorthand certainly makes it easier.